cmake/legacy.cmake

```diff
@@ -351,6 +368,8 @@ ExternalProject_Add(root
   COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/root/fix_tpython_python_3_11_deprecation.patch"
   COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/root/fix_pyroot_cast_error_python_3_11.patch"
   COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/root/fix_root_install_external_tars_with_cmake_3_24.patch"
+  COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/root/fix_compilation_with_gcc13.patch"
+  COMMAND ${patch} -p1 -i "${CMAKE_SOURCE_DIR}/legacy/root/fix_strlcpy_if_glibc_gt_2.38.patch"
```

Comment:
Yes I did consider this but finally decided against this solution. The simply reason for the decision was that CBM did a large check based on this version. Bumping ROOT may, as you already mentioned, introduce new problems. Bumping would also mean for CBM to recheck everything for the release which I wanted to avoid.

Reply:
Consider documenting this in the git commit body.